Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://github.com/SecondSonConsulting/Renew/releases</string>
+	
+	<key>pfm_documentation_url</key>
+	<string>https://github.com/SecondSonConsulting/Renew/wiki</string>
+	
+	<key>pfm_targets</key>
+	<array>
+		<string>system</string>
+	</array>
+	
+	<key>pfm_platforms</key>
+	<array>
+		<string>macOS</string>
+	</array>
+	
+	<key>pfm_description</key>
+	<string>Renew Configuration</string>
+	
+	<key>pfm_domain</key>
+	<string>com.secondsonconsulting.renew</string>
+	
+	<key>pfm_format_version</key>
+	<integer>1</integer>
+	
+	<key>pfm_last_modified</key>
+	<date>2022-11-25T00:00:00Z</date>
+	
+	<key>pfm_title</key>
+	<string>Renew - Second Son Consulting</string>
+	
+	<key>pfm_unique</key>
+	<true/>
+	
+	<key>pfm_version</key>
+	<integer>1</integer>
```

Comment:
Let's sort these alphabetically, these should go on the bottom below the `pfm_subkeys` array

Reply:
Completed, thanks for your review.